torchaudio/functional.py

```diff
@@ -475,51 +478,83 @@ def phase_vocoder(
         >>> x = phase_vocoder(complex_specgrams, rate, phase_advance)
         >>> x.shape # with 231 == ceil(300 / 1.3)
         torch.Size([2, 1025, 231, 2])
-    """
 
-    # pack batch
+    Example - New API (using tensors with complex dtype)
+        >>> freq, hop_length = 1025, 512
+        >>> # (channel, freq, time)
+        >>> complex_specgrams = torch.randn(2, freq, 300, dtype=torch.cfloat)
+        >>> rate = 1.3 # Speed up by 30%
+        >>> phase_advance = torch.linspace(
+        >>>    0, math.pi * hop_length, freq)[..., None]
+        >>> x = phase_vocoder(complex_specgrams, rate, phase_advance)
+        >>> x.shape # with 231 == ceil(300 / 1.3)
+        torch.Size([2, 1025, 231])
+    """
+    USE_COMPLEX = complex_specgrams.is_complex()
```

Comment:
I probably wouldn't actually all caps this, typically that means you have a global constant but that's not what this variable is.

Reply:
yeah makes sense! updated